src/LibFTD2XX.jl

```diff
@@ -200,19 +200,30 @@ function status(handle::FT_HANDLE)
   mflaglist, lflaglist
 end
 
-function Base.nb_available(handle::FT_HANDLE)
-  nbrx = Ref{DWORD}()
-  status = ccall(cfunc[:FT_GetQueueStatus], cdecl, FT_STATUS, 
-                 (FT_HANDLE, Ref{DWORD}),
-                  handle,    nbrx)
-  FT_STATUS_ENUM(status) == FT_OK || throw(FT_STATUS_ENUM(status))
-  nbrx[]
+if VERSION < v"0.7"
```

Comment:
Is this required? Surely `Compat` will map this name to `bytesavailable`, including the new method?

Reply:
`Base.bytesavailable` was not in name space on v0.6, needed `Compat.bytesavailable` to work.